activesupport/lib/active_support/i18n_railtie.rb

```diff
@@ -53,7 +73,18 @@ def self.initialize_i18n(app)
       # Restore available locales check so it will take place from now on.
       I18n.enforce_available_locales = enforce_available_locales
 
-      reloader = ActiveSupport::FileUpdateChecker.new(I18n.load_path.dup){ I18n.reload! }
+      watched_dirs = Helpers.compile_watched_dirs(reloadable_paths)
```

Comment:
I wonder if `watched` is the appropriate word here. Aren't they first `watched` passed to the file update checker?

I'd just go with `directories`.